backend/ops_api/ops/schemas/cans.py

```diff
@@ -101,16 +101,30 @@ class CreateUpdateFundingBudgetSchema(Schema):
     notes = fields.String(load_default=None)
 
 
+class CreateUpdateFundingDetailsSchema(Schema):
```

Comment:
are there any openApi schema changes needed?

Reply:
There are, Frank. Good callout. I will get these.

Reply:
Just fixed this by adding the endpoints and types.

Reply:
thx @rajohnson90